torch/_dynamo/variables/builtin.py

```diff
@@ -1240,6 +1240,16 @@ def call_setattr(
                         return getattr_var
 
             obj.convert_to_unspecialized(tx)
+        # FIXME (tmanlaibaatar) this is utter hack to unblock HuggingFace export
+        # Export generally doesn't want to allow mutations on objects directly,
+        # but we don't have good way to do this rn. For now, we make it an undefined
+        # behaviour and just set attributes directly on the PretrainedConfig object
+        # for now.
+        elif isinstance(obj, variables.dicts.HFPretrainedConfigVariable) and tx.export:
+            if name_var.is_python_constant() and isinstance(
+                val, variables.ConstantVariable
+            ):
+                return obj.var_setattr(tx, name_var, val)
```

Comment:
why not inline the contents of that method right here? Otherwise you need to add the warning comment to that method as well....